### ISSUE TITLE
Can't get Focus Borders working on Windows 11

### ISSUE THREAD

**Issue by DarylGraves** [glazerdesktop/GlazeWM#429]
Hi, sorry if this is a stupid question but I can't get Focus Borders working. 

I have added this at the very top of my config.yaml file:

```
# Focus borders (This doesn't seem to be working)
focus_borders:
  active:
    enabled: true
    color: "#ED7000"
  inactive:
    enabled: false
    color: "#0000ff"

gaps:
  inner_gap: "5px"
  outer_gap: "5px 0 5px 0"

```
Gaps is working but I can't seem to get focus borders to work at all.

I am on the latest version of Windows 11 Pro, Version 22H2 (OS build	22621.2428) so think I should be good. 
Is there a specific place in the config file you should add it? The above extract is line 1.

Thanks!

**Comment by azinsharaf** [comment]
what is you change the color? This one works for me on Win 11.

```
focus_borders:
  active:
    enabled: true
    color: "#ff0000"
  inactive:
    enabled: false
    color: "#0000ff"

gaps:
  # Gap between adjacent windows.
  inner_gap: "30px"
```

**Comment by DarylGraves** [comment]
Thank you for getting back to me! I've copied and pasted yours in but it doesn't seem to work. The borders are big but I can't see any colour between them focused or unfocused, just my normal wallpaper background. Unless I'm just misunderstanding what it does?
![image](https://github.com/glazerdesktop/GlazeWM/assets/46203429/8f01cb64-1ae2-432d-8d14-01430edee02f)

**Comment by DarylGraves** [comment]
This appears resolved in 2.0.3.
